app/decorators/models/solidus_virtual_gift_card/spree/store_credit_category_decorator.rb

```diff
@@ -3,10 +3,10 @@
 module SolidusVirtualGiftCard
   module Spree
     module StoreCreditCategoryDecorator
-      GIFT_CARD_CATEGORY_NAME = 'Gift Card'
-
       def self.prepended(base)
-        base.non_expiring_credit_types |= [GIFT_CARD_CATEGORY_NAME]
+        gift_card_category_name = 'Gift Card'
+        base.const_set(:GIFT_CARD_CATEGORY_NAME, gift_card_category_name) unless base.const_defined?(:GIFT_CARD_CATEGORY_NAME)
```

Comment:
Thanks! 😄 Every time I start solidus in console for the past 4 years I have seen this duplicated constant error 😅